mailpoet/assets/js/src/common/set_from_address_modal.tsx

```diff
@@ -79,7 +79,7 @@ type Props = {
 };
 
 function SetFromAddressModal({ onRequestClose, setAuthorizedAddress }: Props) {
-  const [address, setAddress] = useState('');
+  const [address, setAddress] = useState<string>();
```

Comment:
Note that replacing the `('')` with `()` actually changes behavior – now it's `string | undefined` while previously it was `string` (only not typed). Not sure if it can break something.

Reply:
Ahh, only now I see this was originally `useState(null);` – so we're just changing the default value from `null` to `undefined`, which should be fine 👍